packages/headless/src/features/advanced-search-queries/advanced-search-queries-state.ts

```diff
@@ -1,16 +1,53 @@
+export interface AdvancedSearchQueriesDefaultFiltersState {
+  /**
+   * The initial cq filter (e.g., `((q AND aq) OR dq) AND cq).
+   */
+  cq: string;
+
+  /**
+   * The initial aq filter (e.g., `((q AND aq) OR dq) AND cq).
+   */
+  aq: string;
```

Comment:
I added support to `aq` as well since it makes the behaviour of `AdvancedSearchQueries` more consistent across both, therefor less confusing. If you prefer, I can remove support for the default value for `aq`.